views/account/org/edit-question.jade

```diff
@@ -0,0 +1,18 @@
+extends ../../app_layout
+
+block content
+ .col.s10.offset-s1.m8.offset-m2.l6.offset-l3
+    h3 Edit Question
+    .row
+      .card.col.s10
+        h4 Question #
+        .row
+          form.form-horizontal.col.s10(id='edit-question-form', method='POST')
+            input(type='hidden', name='_csrf', value=_csrf)
+            .row
+              .input-field.col.s10
+                input.validate(type='text', name="question", id='question', value = "Previous Question should be in here")
+            .row
+              a(href="/edit-survey")
+                img(src="https://cdn2.iconfinder.com/data/icons/free-basic-icon-set-2/300/17-512.png" width="40px" align="left")
```

Comment:
Done donedoneand donethanks for the feedback, I'll keep it in mind when further coding  

    On Saturday, February 6, 2016 7:21 PM, DylanMoz <notifications@github.com> wrote:
 
 

 In views/account/org/edit-question.jade:> +
> +block content
> + .col.s10.offset-s1.m8.offset-m2.l6.offset-l3
> +    h3 Edit Question
> +    .row
> +      .card.col.s10
> +        h4 Question #
> +        .row
> +          form.form-horizontal.col.s10(id='edit-question-form', method='POST')
> +            input(type='hidden', name='_csrf', value=_csrf)
> +            .row
> +              .input-field.col.s10
> +                input.validate(type='text', name="question", id='question', value = "Previous Question should be in here")
> +            .row
> +              a(href="/edit-survey")
> +                img(src="https://cdn2.iconfinder.com/data/icons/free-basic-icon-set-2/300/17-512.png" width="40px" align="left")
You can use font-awesome, an icon library we are using, instead of these images. Look at the home page under the area you worked on last week for an example—
Reply to this email directly or view it on GitHub.